src/identities/mnemonic.ts

```diff
@@ -24,7 +24,7 @@ interface AddressInterfaceExtended {
   derivationPath: string;
 }
 
-type AddressCacheMap = Record<string, AddressInterfaceExtended>;
+type AddressCacheMap = { [script: string]: AddressInterfaceExtended };
```

Comment:
I've tried Record, Map, and a typed object: all of these methods return undefined for string's key. It seems that Map, Record, and object use '===' to find the key's value. When the key is a Buffer or a string, it leads to very weird problems... I'm searching solution

Reply:
Solved using `String.ValueOf()`

Reply:
hey normal js objcet would be more than enough :)